modules/Template/PortalPublication.yaml

```diff
@@ -0,0 +1,70 @@
+#######################################
+# Portal          
+#######################################
+
+# Note: this template definition uses the alternate syntax with 'attribute'
+
+classes:
+  PublicationTemplate:
+    is_a: Template
+    description: >
+      A generalized data model template for Publications to be hopefully used across dccs
+    notes:
+    - This structure is based on a CSV file from MC2 annotations with customized fields as attributes.
+    attributes:
+      grantNumber:
+        range: string
+        required: true
+      consortiumName:
+        range: string
+        required: true
+      themeName:
+        range: string
+        required: true
+      doi:
+        range: string
+        required: true
+      journal:
+        range: string
+        required: true
+      pmid:
+        range: integer
+        required: true
+      pubmedUrl:
+        range: string
+        required: true
+      title:
+        range: string
+        required: true
+      year:
+        range: integer
+        required: true
+      diseaseFocus:
+        range: string
+        multivalued: true
+        required: false
+      authors:
+        range: string
+        multivalued: true
+        required: false
+      abstract:
+        range: string
+        required: true
+      assay:
+        range: string
+        multivalued: true
+        required: false
+      accessibility:
+        range: string
+        required: true
+
+  NFPublication:
```

Comment:
Can you change name of class to `PortalPublication` to match the rest of the portal entities (`PortalStudy`, etc.)?